### ISSUE TITLE
Order Tax Rate Information not saved  - Full Tax Summary not working

### ISSUE THREAD

**Issue by fooman** [magento/magento2#4906]
## Steps to reproduce
1. Install Magento 2.1.0-rc1
2. Create the following tax rates and rule
   ![screenshot-8b42285d9d75 local fooman co nz 2016-06-05 23-46-57](https://cloud.githubusercontent.com/assets/455508/15808433/dab0d0aa-2b77-11e6-9fa9-a8941b372431.png)

![screenshot-8b42285d9d75 local fooman co nz 2016-06-05 23-41-31](https://cloud.githubusercontent.com/assets/455508/15808426/a5a5ba74-2b77-11e6-963a-ded1a232d7ac.png)
1. Place an order in New York

![screenshot-8b42285d9d75 local fooman co nz 2016-06-05 23-41-53](https://cloud.githubusercontent.com/assets/455508/15808459/ea39fdc6-2b77-11e6-9272-714457a2543e.png)
1. Enable Stores > Configuration > Tax > Orders, Invoices, etc > Display Full Tax Summary

![screenshot-8b42285d9d75 local fooman co nz 2016-06-05 23-40-38](https://cloud.githubusercontent.com/assets/455508/15808466/21c20112-2b78-11e6-94a4-82273467bc16.png)
## Expected result
1. Tax breakdown by tax rate is displayed
## Actual result

![screenshot-8b42285d9d75 local fooman co nz 2016-06-05 23-42-18](https://cloud.githubusercontent.com/assets/455508/15808465/0e632740-2b78-11e6-8259-71a9e9cfe125.png)
on clicking the tax row

Also not looking great in the database:

```
mysql> select * from sales_order_tax_item;
Empty set (0.00 sec)

mysql> select * from sales_order_tax;
Empty set (0.00 sec)
```

**Comment by mikeweis** [comment]
@fooman Thank you for reporting. We have created internal ticket MAGETWO-53005 to track this issue.

**Comment by magento-engcom-team** [comment]
@fooman, thank you for your report.
The issue is already fixed in 2.2.0
